internal/pkg/internal/utils_test.go

```diff
@@ -0,0 +1,23 @@
+package internal
+
+import (
+	"mittens/internal/pkg/probe"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateTempFile(t *testing.T) {
+	var content string = "testfile"
+	var generatedFile string = CreateTempFile(content)
+
+	// check if the file generated is exist
+	var expected bool = true
+	fileCheckBool, err := probe.FileExists(generatedFile)
+	require.NoError(t, err)
+
+	// clean up test file
+	probe.DeleteFile(generatedFile)
+	assert.Equal(t, expected, fileCheckBool)
```

Comment:
thanks for the contribution :)

a bit of nitpicking but do you mind moving this line (assert) a bit up? i think it would make sense to have this after the require.NoError as it's related to the assertion that the file was created and unrelated to the cleanup.

Reply:
I made the changes, please kindly check. thank you